From c3f816afb3dcecefe6087c30093c500fb80acaab Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Mon, 3 Mar 2025 11:38:19 +0100
Subject: [PATCH 1654/1695] plat-stm32mp1: drop prototype of
 stm32_cores_reset()

The function stm32_cores_reset() has never been implemented.
Drop the prototype.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c4ff73ac0d3580898927d3859f195a6abd768894 plat-stm32mp1: drop prototype of stm32_cores_reset()]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I7afbd38c330c82611675e987ebd48b5b84eb4eee
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/441309
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/arch/arm/plat-stm32mp1/stm32mp_pm.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/core/arch/arm/plat-stm32mp1/stm32mp_pm.h b/core/arch/arm/plat-stm32mp1/stm32mp_pm.h
index 291ad5d1a..a4dcb2292 100644
--- a/core/arch/arm/plat-stm32mp1/stm32mp_pm.h
+++ b/core/arch/arm/plat-stm32mp1/stm32mp_pm.h
@@ -15,8 +15,6 @@ enum stm32mp1_pm_domain {
 	STM32MP1_PD_MAX_PM_DOMAIN
 };
 
-void __noreturn stm32_cores_reset(void);
-
 /* Convert low power mode carried by pm_hint into pm mode name */
 const char *stm32mp_pm_hint2mode_name(uint32_t pm_hint);
 
-- 
2.47.2

